client/my-sites/plugins/no-permissions-error.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import EmptyContent from 'components/empty-content';
+import DocumentHead from 'components/data/document-head';
+import SidebarNavigation from 'my-sites/sidebar-navigation';
+
+function NoPermissionsError( { translate } ) {
+	return (
+			<Main>
+				<DocumentHead title={ translate( 'Plugins' ) } />
```

Comment:
I don't think this `DocumentHead` title makes sense in all cases. Looking at #15482 I can see you're using this component in the plugins main page, but also in the plugins browser and in the single plugin page. I'd suggest staying consistent and not introducing any changes with the page titles. So, we'll need a separate title in all of these cases - I suggest that you make sure to check out the current title in these cases, and apply it accordingly.